README.md

```diff
@@ -81,7 +87,152 @@ credentials for third-party services manually in order to run the development
 server or the integration tests.
 
 The environment variables in `.env` customize the settings from
-`opencraft/settings.py` which are loaded via `env()`.
+`opencraft/settings.py` which are loaded via `env()`. For more information about
+each setting, see [`opencraft/settings.py`](opencraft/settings.py). As a
+minimum, you will need to add credentials for OpenStack, Gandi and GitHub. You
+can use this example `.env` file as a starting point:
+
+```sh
+SECRET_KEY='...'
+DATABASE_URL='postgres://localhost/opencraft'
+OPENSTACK_USER='username'
+OPENSTACK_PASSWORD='password'
+OPENSTACK_TENANT='tenant-name'
+OPENSTACK_AUTH_URL='https://auth.cloud.ovh.net/v2.0'
+OPENSTACK_REGION='BHS1'
+OPENSTACK_SANDBOX_SSH_KEYNAME='keypair-name'
+INSTANCES_BASE_DOMAIN='example.com'
+GANDI_ZONE_ID='123456789'
+GANDI_API_KEY='api-key'
+GITHUB_ACCESS_TOKEN='github-token'
+WATCH_ORGANIZATION='github-org'
+```
+
+### A note on SSH keys
+
+The instance manager uses [Ansible](https://www.ansible.com/) to provision
+openedx sandboxes. Ansible uses SSH to run commands on remote servers, so you
+will need to configure the instance manager with an SSH key pair to use for this
+purpose.
+
+First, **as the user that the instance manager runs as**, run:
```

Comment:
@omarkhan Does this mean I also need to create a new user inside my vagrant instance to run the instance manager, or is one already created for me?  If not, that should be part of the vagrant setup instructions (with the local install setup, it's clear enough already).